### DISCUSSION TITLE
Color.WHITE should be added to ColorDistanceSensor

### DISCUSSION THREAD

**Discussion by EclecticInventor** [pybricks/support#1317] [Ideas]
The Color.WHITE should be added to ColorDistanceSensor, as an option for the light.on command (turning on RGB, making white)

**Comment by laurensvalk** [comment]
The sensor doesn't support that, unfortunately.

The closest you can get is by putting it in color mode (just call color() to do so), but you'll still see distinct colors. 

Or do you mean that this should happen automatically when using Color.WHITE?

**Reply by EclecticInventor** [reply]
"Or do you mean that this should happen automatically when using Color.WHITE?" 
Yes, that's what I mean it should do. It's close enough to white.
